pumpkin/src/server/mod.rs

```diff
@@ -163,6 +164,21 @@ impl Server {
         None
     }
 
+    /// Get the player count sum in all worlds
+    pub async fn get_player_count(&self) -> u32 {
+        let mut count = 0;
+        for world in &self.worlds {
+            count += world
+                .current_players
+                .lock()
+                .await
+                .len()
+                .to_u32()
+                .expect("Unable to convert to u32");
+        }
+        count
+    }
```

Comment:
you don't need to convert it to u32, just use usize

Reply:
`BASIC_CONFIG.max_players` is `u32` which is why I used the conversion, also `usize` is platform dependent , please let me know what you think

Reply:
So generally we want to use usize for such things like a player count. I also want to give the option to disable a player limit, maybe by setting it to -1 ?. the config value can stay u32 because this will be more than enough if someone wants to set a limit

Reply:
TL;DR return usize. If you compare it with the config value, cast it

Reply:
Cheers - should be resolved in 8aba1ff. I've also added logic that setting the max players to zero will disable the limit. (since it's unsigned, we can't use `-1`)